solaris/solaris11/SPECS/template_agent_v4.2.0.json

```diff
@@ -1586,399 +1586,399 @@
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/cve-ubuntu-xenial-oval.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-centos-6-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-centos-7-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-debian-8-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-fedora-24-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-rhel-6-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-rhel-7-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-ubuntu-1404-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/content/ssg-ubuntu-1604-ds.xml": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0640",
         "prot": "-rw-r-----",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/oscap.py": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0750",
         "prot": "-rwxr-x---",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/template_oval.xsl": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
         "mode": "0750",
         "prot": "-rwxr-x---",
         "type": "file",
         "user": "root"
     },
     "/var/ossec/wodles/oscap/template_xccdf.xsl": {
         "class": "static",
-        "group": "ossec",
+        "group": "wazuh",
```

Comment:
You forgot to update `template_agent_v5.0.0.json`.

Reply:
Done 1034578